docs/resources/v2-conformance-statement.md

```diff
@@ -75,21 +76,27 @@ The following DICOM elements are required to be present in every DICOM file atte
 
 Each file stored must have a unique combination of StudyInstanceUID, SeriesInstanceUID and SopInstanceUID. The warning code `45070` will be returned if a file with the same identifiers already exists.
 
-> DICOM File Size Limit: there is a size limit of 2GB for a DICOM file by default.
+> Requests are limited to 2GB. No single DICOM file or combination of files may exceed this limit.
+
+### Store Changes From V1
+A major change from V1 Store API is that attributes will continue to be validated, but only validation errors with [required attributes](#store-required-attributes) would cause for Store to fail.
```

Comment:
Minor suggestion: 

```suggestion
A major change from V1 Store API is that attributes will continue to be validated, but only validation errors with [required attributes](#store-required-attributes) would cause Store to fail.
```